packages/plugin-dts/README.md

```diff
@@ -2,18 +2,169 @@
   <img alt="Rslib Banner" src="https://assets.rspack.dev/rslib/rslib-banner.png">
 </picture>
 
-# Rslib
+# rsbuild-plugin-dts
 
-Rslib is a library development tool powered by [Rsbuild](https://rsbuild.dev). It allows library developers to leverage the knowledge and ecosystem of webpack and Rspack.
+An Rsbuild plugin to emit declaration files for TypeScript which is built-in in Rslib.
 
-## Documentation
+## Using in Rslib
 
-https://lib.rsbuild.dev/
+Read [DTS](https://lib.rsbuild.dev/guide/advanced/dts) and [lib.dts](https://lib.rsbuild.dev/config/lib/dts) for more details.
+
+## Using in Rsbuild
+
+Install:
+
+```bash
+npm add rsbuild-plugin-dts -D
+```
+
+Add plugin to your `rsbuild.config.ts`:
+
+```ts
+// rsbuild.config.ts
+import { pluginDts } from 'rsbuild-plugin-dts';
+
+export default {
+  plugins: [pluginDts()],
+};
+```
+
+## Options
+
+### bundle
+
+- **Type:** `boolean`
+- **Default:** `false`
+
+Whether to bundle the DTS files.
+
+If you want to generate [bundle DTS](https://lib.rsbuild.dev/guide/advanced/dts#bundle-dts) files, you should:
+
+- Install `@microsoft/api-extractor` as a development dependency.
+
+```bash
+npm add @microsoft/api-extractor -D
+```
+
+- Set `bundle` to `true`.
+
+```js
+pluginDts({
+  bundle: true,
+});
+```
+
+### distPath
+
+- **Type:** `string`
+
+The output directory of DTS files. The default value follows the priority below:
+
+1. The `distPath` value of the plugin options.
+2. The `declarationDir` value in the `tsconfig.json` file.
+3. The [output.distPath.root](https://rsbuild.dev/config/output/dist-path) value of Rsbuild configuration.
+
+```js
+pluginDts({
+  distPath: './dist-types',
+});
+```
+
+### build
+
+- **Type:** `boolean`
+- **Default:** `false`
+
+Determines whether to generate DTS files while building the project's references. This is equivalent to using the `--build` flag with the `tsc` command.
+
+When this option is enabled, you must explicitly set `declarationDir` or `outDir` in `tsconfig.json` in order to meet the build requirements.
+
+### abortOnError
+
+- **Type:** `boolean`
+- **Default:** `true`
+
+Whether to abort the build process when an error occurs during DTS generation.
+
+By default, type errors will cause the build to fail. When `abortOnError` is set to `false`, the build will still succeed even if there are type issues in the code.
+
+```js
```

Comment:
Adding ``For example, disable `abortOnError` to allow type error.``. The following fields also need to update.